src/components/custom-antd/AutoComplete/AutoComplete.tsx

```diff
@@ -0,0 +1,7 @@
+import {AntdCustomStyledAutoComplete, ICustomAutoCompleteProps} from './AutoComplete.styled';
+
+const AutoComplete: React.FC<ICustomAutoCompleteProps> = props => {
+  return <AntdCustomStyledAutoComplete {...props} />;
+};
```

Comment:
Will it be used later, or should we just export the styled component directly instead?